### PR TITLE
added timestamps to asa_command module

### PR DESCRIPTION
**Pull request by vaneuk**
asa_command module now returns timestamps field, which shows command execution time

##### SUMMARY

Added information about time when command was issued on remote device.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

asa_command

##### ADDITIONAL INFORMATION

**example playbook**
```yaml
- name: collect show commands from devices
  hosts: lab_asa
  connection: network_cli

  tasks:

    - name: collect show commands
      asa_command:
        commands:
          - "show clock"
          - "show context count"
      register: output

    - name: debug task
      debug:
        var: output
```

**before**
```
ok: [ASA-act] => {
    "output": {
        "changed": false, 
        "failed": false, 
        "stdout": [
            "16:15:10.329 MSK Wed Dec 19 2018", 
            "Total active Security Contexts: 9"
        ], 
        "stdout_lines": [
            [
                "16:15:10.329 MSK Wed Dec 19 2018"
            ], 
            [
                "Total active Security Contexts: 9"
            ]
        ]
    }
}
```

**after**
```
ok: [ASA-act] => {
    "output": {
        "changed": false, 
        "failed": false, 
        "stdout": [
            "16:21:17.639 MSK Wed Dec 19 2018", 
            "Total active Security Contexts: 9"
        ], 
        "stdout_lines": [
            [
                "16:21:17.639 MSK Wed Dec 19 2018"
            ], 
            [
                "Total active Security Contexts: 9"
            ]
        ], 
        "timestamps": [
            "2018-12-19T16:21:31", 
            "2018-12-19T16:21:31"
        ]
    }
}
```

### CODE REVIEW

File: lib/ansible/module_utils/basic.py [hunk 1]
```diff
@@ -85,6 +85,7 @@
 import datetime
 from collections import deque
 from itertools import chain, repeat
+from datetime import datetime
```
Comment (mattclay) [inline lib/ansible/module_utils/basic.py]: This conflicts with the `import datetime` above.

Comment (vaneuk) [reply lib/ansible/module_utils/basic.py]: thanks, fixed

**Comment by vaneuk** [comment]
@gdpak @ogenstad @privateip could you kindly take a look at this?

**Comment by vaneuk** [comment]
@gdpak @ogenstad @privateip could you kindly take a look at this?

**Comment by vaneuk** [comment]
@mattclay @gdpak @ogenstad @privateip could you kindly take a look at this?

**Review by ganeshrn — COMMENTED** [review]
The timestamp information can be extracted in playbook from `ansible_date_time` facts value along with ansible loop to figure out timestamp for each command. Adding it as a parameter to control the return of timestamp information to cliconf run_command() api (for ios, iosxr and nxos) does not seem correct

**Comment by vaneuk** [comment]
@ganeshrn could you kindly provide an example of such task? I had spent some time trying to figure out how this can be achieved using ansible built-in methods before created the PRs.
I find it convenient and logical to have timestamp of each command in run_commands().

**Comment by vaneuk** [comment]
Hi @ganeshrn 
did you have a chance to take a look at it once more?

**Comment by vaneuk** [comment]
Hi @ganeshrn 
Could you kindly take a look at this?

**Comment by danielmellado** [comment]
Thank you very much for your interest in Ansible. This plugin is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/cisco.asa
Please re-submit this PR in the above repository.
If you have further questions please stop by IRC or the mailing list:

IRC: #ansible-network on irc.freenode.net
mailing list: https://groups.google.com/forum/#!forum/ansible-project

**Pull request closed without merging**